### DISCUSSION TITLE
Deployments have different result

### DISCUSSION THREAD

**Discussion by ebertulfo** [vercel/vercel#4192] [General]
I'm deploying a NextJS in Now and I've been having trouble with the deployment.

I did the deployment in 3 different envs, the plain `now` deployment, `now staging`, and `now production`.

For Staging, the styles is bugged. But for the plain `now` and `now production` deployments, it works perfectly fine.

Did someone experienced this kind of issue?

**Comment by paulogdm** [comment]
It is hard to debug such behavior without a reproducible code. Can you provide one?
